docusaurus.config.js

```diff
@@ -194,14 +194,10 @@ const config = {
           },
         ],
       },
-      algolia: {
-        appId: 'DMPMC33PLU',
-        apiKey: '12d46b3bfeee6511031cfe00778f3e45',
-        indexName: 'fastify',
-      },
     }),
 
   plugins: [
+    '@orama/plugin-docusaurus',
```

Comment:
Was looking to turn on the hotkey: 

<img width="187" alt="image" src="https://github.com/fastify/website-next/assets/11404065/d6df48ae-8933-46f1-8bf9-b8dda453800e">

Do you know how?
(in case we may open a new dedicated issue)

Reply:
To be honest now, this is on the Algolia autocomplete plugin.
If you don't find a solution before me, I'll take a look on Monday.

Reply:
I just found this bug that show us where is the code
https://github.com/algolia/docsearch/pull/1590/files